### ISSUE TITLE
Godot Namespace Not Found and Can't Run Scene

### ISSUE THREAD

**Issue by Rybadour** [GodotExplorer/godot-tools#91]
VisualStudio code seems to have the right packages installed and I've configured the correct path for the GodotEditor but all of the scripts in my existing project say that the Godot namespace is not found. Also when I try to run the scene it open Godot but fails. Seems with this:

```
ERROR: Failed loading scene: res://Tileset_Env.cs
   At: main/main.cpp:1688
Mono: Runtime cleanup...
debugger-agent: Unable to listen on 1832
WARNING: cleanup: ObjectDB Instances still exist!
     At: core/object.cpp:1989
ERROR: clear: Resources Still in use at Exit!
   At: core/resource.cpp:418
```

Did I forgot part of the setup?

**Comment by Geequlim** [comment]
This plugin is designed to write GDScript easier.
It does nothing for C# related stuff.

**Comment by Rybadour** [comment]
Update:

I haven't fixed the second issue but I fixed the first by running the scene once in Godot. I guess I must have deleted what was in the .mono/assemblies directory because once I ran it in Godot it copied some dlls over.

> This plugin is designed to write GDScript easier.
It does nothing for C# related stuff.

Sorry it wasn't clear to me. Even the second problem does sound like an issue with Mono or whatever instead. I'll close this issue.
